pkgs/by-name/aw/aws-gate/disable-bootstrap.patch

```diff
@@ -0,0 +1,316 @@
+diff --git a/aws_gate/bootstrap.py b/aws_gate/bootstrap.py
+deleted file mode 100644
+index 89e7516..0000000
+--- a/aws_gate/bootstrap.py
++++ /dev/null
```

Comment:
I'd probably not include the file deletions in the patch but rather as `postPatch` in `mkDerivation` like suggested below, because every minor upstream change in these fails will cause the patch to not apply anymore.

Reply:
That would indeed have been very unpleasant. Fixed!